src/components/Hero/Hero.stories.tsx

```diff
@@ -371,6 +372,48 @@ export const CampaignBackgroundColors: Story = {
   ),
 };
 
+const FallBackExample = () => {
+  const [imageSrc, setImageSrc] = useState("foobar.jpg");
+  const fallbackImageSrc =
+    "https://loremflickr.com/cache/resized/65535_51121136167_40f20af042_z_300_300_nofilter.jpg";
```

Comment:
This image path may expire.  I had an image from loremflickr.com open in a browser tab for a day or two and eventually I was unable to reload the image, leading me to believe the path had expired.  It worries me that `cache` is part of the path.